### DISCUSSION TITLE
There should be option for whether user wants to reset the default styles or not

### DISCUSSION THREAD

**Discussion by indahud** [tailwindlabs/tailwindcss#4362] [Ideas]
There should be option for whether user wants to reset the default styles or not. Sometimes default styles are needed in some cases. For example I am using `dangerouslySetInnerHTML={{ __html: content }}` to fetch the HTML data. Typography plugin is solving some of my issues but design is kind of breaking in `p` , `span` and `code` tags. For those reason I don't want to reset default styles. And don't want to manually add classnames to all the HTML.

**Comment by dance2die** [comment]
You can set `preflight` to false to disable style resets: https://tailwindcss.com/docs/preflight#disabling-preflight

It's one of the core Tailwind CSS plugins, which you can learn more about here: https://tailwindcss.com/docs/configuration#core-plugins
